Apps/phvmray/vmray.json

```diff
@@ -86,139 +84,201 @@
+                    ],
+                    "data_path": "action_result.summary.vault_id",
                     "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
                 }
-            ]
+            ],
+            "render": {
+                "type": "table",
+                "width": 2,
+                "height": 5,
+                "title": "Get File Details"
+            },
+            "versions": "EQ(*)"
         },
         {
             "action": "detonate file",
-            "description": "Detonate file in the VMRay Analyzer",
-            "verbose": "The <b>file_name</b> parameter overrides the filename, if none is given the app tries to get the filename from the vaults metadata. The <b>type</b> overrides the automatic detection of the VMRay Analyzer. The <b>config</b> parameter specifies additional configuration options passed to the Analyzer (See user_config in the REST API documentation). With <b>jobrules</b> you can specify custom jobrule entries (See jobrule_enries in the REST API documentation).",
-            "type": "investigate",
+            "description": "Detonate file in the VMRay Platform",
+            "verbose": "The <b>file_name</b> parameter overrides the filename, if none is given the app tries to get the filename from the vaults metadata. The <b>type</b> overrides the automatic detection of the VMRay Platform. The <b>config</b> parameter specifies additional configuration options passed to the VMRay Platform (See user_config in the REST API documentation). With <b>jobrules</b> you can specify custom jobrule entries (See jobrule_enries in the REST API documentation).",
+            "type": "generic",
             "identifier": "detonate_file",
             "read_only": false,
-            "versions": "EQ(*)",
-            "render": {
-              "type": "table",
-              "width": 3,
-              "height": 5,
-              "title": "Detonate File Details"
-            },
-            "lock": {
-                "enabled": true,
-                "data_path": "parameters.vault_id",
-                "timeout": 300
-            },
             "parameters": {
                 "vault_id": {
-                    "description": "The vault_id of the file to be analyzed.",
+                    "description": "The vault_id of the file to be analyzed",
                     "data_type": "string",
-                    "contains": ["vault id", "pe file", "pdf", "flash", "apk", "jar", "doc", "xls", "ppt", "javascript"],
+                    "contains": [
```

Comment:
@jthom-vmray, could you please confirm for these contains as this parameter is expecting vault id? I guess, it only needs "vault id" contains and others should be either _example values_ or _value list_ for the **type** action parameter for this same action.

Reply:
Yes, you are right. `vault_id` is the only parameter expected here.